### PR TITLE
Add a comment for libpng upgrade

### PR DESCRIPTION
**Pull request by vrabaud**

### CODE REVIEW

**Review by wantehchang — APPROVED** [review]

File: cmake/Modules/LocalZlibpng.cmake [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# To upgrade libpng to > v1.6.40, you need zlib containing f1f503da85d52e56aae11557b4d79a42bcaa2b86
+# hence a version > v1.3 .
```
[the remark this answers is not in the document]
Comment (wantehchang) [reply cmake/Modules/LocalZlibpng.cmake:2]: I found the answer. This comment has a typo. It should say v1.3.1, not v1.3.

File: cmake/Modules/LocalZlibpng.cmake [hunk 2]
```diff
@@ -56,6 +58,11 @@ set(CMAKE_DEBUG_POSTFIX "")
 
 add_library(ZLIB::ZLIB ALIAS zlibstatic)
 
+# TODO Uncomment for zlib > v1.3
+# install(TARGETS zlib zlibstatic EXPORT zlib)
+# install(EXPORT zlib
+#         DESTINATION ${CMAKE_INSTALL_LIBDIR}/zlib)
```
[the remark this answers is not in the document]
Comment (wantehchang) [reply cmake/Modules/LocalZlibpng.cmake:64]: Can we upgrade to zlib v1.3.1 now?

File: cmake/Modules/LocalZlibpng.cmake (hunk 1 above)
[the remark this answers is not in the document]
Comment (vrabaud) [reply cmake/Modules/LocalZlibpng.cmake]: Indeed, just fixed it.

File: cmake/Modules/LocalZlibpng.cmake (hunk 2 above)
[the remark this answers is not in the document]
Comment (vrabaud) [reply cmake/Modules/LocalZlibpng.cmake:64]: Actually no: the first install basically says: put the zlib target in the export group name zlib.
The second one creates and installs the files related to the export group zlib.

[the remark this answers is not in the document]
Comment (wantehchang) [reply cmake/Modules/LocalZlibpng.cmake:64]: Thank you. I studied the [documentation of the cmake install() function](https://cmake.org/cmake/help/latest/command/install.html). I understand this better now.

[the remark this answers is not in the document]
Comment (vrabaud) [reply cmake/Modules/LocalZlibpng.cmake:64]: Indeed, we can bump zlib only, cf https://github.com/AOMediaCodec/libavif/pull/2192

**Pull request merged**